firefox_puppeteer/ui/tabbar.py

```diff
@@ -3,126 +3,315 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from marionette import (
-    HTMLElement,
     Wait,
 )
 
-from marionette.errors import NoSuchElementException, StaleElementException
+from marionette.errors import NoSuchElementException
+
+import firefox_puppeteer.errors as errors
 
 from .. import DOMElement
-from ..base import BaseLib
+from ..base import UIBaseLib
 
 
-class Tabs(BaseLib):
+class TabBar(UIBaseLib):
 
-    # TODO arrow scrollers, drop down list
+    # Properties for visual elements of the tabs toolbar #
```

Comment:
Nope, it's just an indicator for code blocks. I will add a doc comment for the class.